### ISSUE TITLE
Обобщенные классы

### ISSUE THREAD

**Issue by sergoio0704** [dmcharlie19/Task3_UniversitySql#4]
https://github.com/dmcharlie19/Task3_UniversitySql/blob/39da09db14781ac916fd3adfa9ac5890a67a8a67/Task3/Repositories/CommonOperationsSqlRepository.cs#L1

Не стоит делать обобщенные репозитории, так как могу возникнуть проблемы, составлении более сложных запросов. Лучше разделить на Student и Group

**Comment by dmcharlie19** [comment]
да, понимаю, модели могут развиваться по разному и в будущем такой репозиторий может приносить больше проблем чем пользы. Изначально было 3 репозитория, для каждой сущности свой. Мне заходилось объединить очень похожий код в одном месте. Опыт показался интересным, изучил инструенты рефлексии.
Предлагаю в данном проектк оставить как есть, он ведь не будет дальше развиваться

**Comment by sergoio0704** [comment]
Хорошо
